internal/locate/region_request.go

```diff
@@ -669,6 +670,7 @@ func newReplicaSelector(regionCache *RegionCache, regionID RegionVerID, req *tik
 		cachedRegion,
 		regionStore,
 		replicas,
+		option,
```

Comment:
Here we make the copy of `option`, and currently we do not use `option.leaderOnly` so it's safe, but I'm worring about the risk, because `replicaSelector.option.leaderOnly` and `accessFollower.option.leaderOnly` may be different.

For me, it's better to make `replicaSelector.option` a reference.

Reply:
Great, done.